Makefile

```diff
@@ -11,17 +9,22 @@ test:
 test-integration:
 	go version
 	terraform --version
+	. ./tf-dev-override.sh
 	go test -run=CLI ./...
 
 test-go-licenses:
 	cd .. && go version && go install github.com/google/go-licenses@latest
 	$$(go env GOPATH)/bin/go-licenses check ./... --ignore github.com/dnaeon/go-vcr
 
-build-docker:
```

Comment:
could you remove this from PHONY as well?

Reply:
Done